### ISSUE TITLE
Install didn't install API subdirectory...

### ISSUE THREAD

**Issue by brbester** [CiscoDevNet/ciscosparkapi#6]
```
 ~/ciscosparkapiexamples $ ls /usr/local/lib/python2.7/dist-packages/ciscosparkapi/
exceptions.py   helper.pyc    restsession.py   sparkdata.pyc
exceptions.pyc  __init__.py   restsession.pyc  _version.py
helper.py       __init__.pyc  sparkdata.py     _version.pyc
```

So when I run your example it can't load ......
```

~/ciscosparkapiexamples $ python example.py                   Traceback (most recent call last):
  File "example.py", line 1, in <module>
    from ciscosparkapi import CiscoSparkAPI
  File "/usr/local/lib/python2.7/dist-packages/ciscosparkapi/__init__.py", line 8, in <module>
    from api.accesstokens import AccessToken, AccessTokensAPI
ImportError: No module named api.accesstokens
```

**Comment by cmlccie** [comment]
This should be fixed now (v0.1.1).  I have have posted the updated package to PyPI.  Please upgrade your local installation and let me know if you have any further package issues.

**Comment by brbester** [comment]
Looks fixed - thanks!
